examples/equilibria/fem_fixed_boundary/fem_magnetostatic_2D_single_coil.py

```diff
@@ -0,0 +1,269 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+
+"""
+Application of the dolfin fem 2D magnetostatic to a single coil problem
+"""
```

Comment:
We could hide some code using the already developed functions for the creation of a PFcoil and the calculation of its analytical B field, implementing a function for the "automatic" creation of the enclosure, and deciding to interface the EM solver also with a component (and not only with the direct mesh). In this way, the example would be "only" focused on the em solver (i.e. 2 line code :) ). However, this would require some more time, so, considering our priority, I would come back to any possible simplification when the fixed boundary module is completed. If it is fine with you, we can add an issue as soon as the PR is merged (just to not forget it).